tests/TicTacToeTest.elm

```diff
@@ -1,19 +1,15 @@
 module TicTacToeTest exposing (..)
 
-import TicTacToe exposing (..)
-import ElmTest exposing (..)
+import Test exposing (..)
+import Expect
 
-tests =
-  suite "TicTacToe Test"
-    [ 
-      test 
-      "" 
-      <| assertEqual 2 (1 + 1)
 
-    ]
+all : Test
+all =
+    describe "TicTacToe Test"
+        [ test "Addition" <|
+            \() ->
+                Expect.equal (3 + 7) 10
```

Comment:
I guess this is to see that the tests work...
I would commit this list empty if no real tests for TicTacToe exist yet and keep this only on my local changes or on another branch...